app/src/main/java/org/wordpress/pioupiou/login/LoginActivity.java

```diff
@@ -224,6 +239,36 @@ public void onAuthenticationChanged(OnAuthenticationChanged event) {
     public void onSiteChanged(OnSiteChanged event) {
         if (!event.isError()) {
             showPostListAndFinish();
+        } else {
+            // TODO
+            Timber.i("onSiteChanged error");
+        }
+    }
+
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onDiscoveryChanged(AccountStore.OnDiscoveryResponse event) {
```

Comment:
Let's add an import for `OnDiscoveryResponse` so this doesn't have the `AccountStore.` part and matches the other OnChanged methods.